bridge/engine/EngineStreamDirector.h

```diff
@@ -69,12 +70,12 @@ class EngineStreamDirector
         if (_participantStreams.find(endpointIdHash) != _participantStreams.end())
         {
             logger::debug("addParticipant stream already added, endpointIdHash %lu",
```

Comment:
@olofkallander, Do we want to keep these explicit logger or change them to DIRECTOR_LOG?